kommunicate/src/main/java/io/kommunicate/KmConversationHelper.java

```diff
@@ -170,7 +176,7 @@ public static void launchChat(final KmChatBuilder launchChat, final KmCallback c
             }
         } else {
             if (!TextUtils.isEmpty(launchChat.getApplicationId())) {
-                Kommunicate.init(launchChat.getContext(), launchChat.getApplicationId());
+                Kommunicate.init(launchChat.getContext(), launchChat.getApplicationId(), KmAppSettingPreferences.getInstance().isRootDetectionEnabled());
```

Comment:
As discussed we don't have option to add these check in `appconfig.json` because we want to block all api calls login/chat creation etc. so in that case we want the `isDeviceRoot` check which is present is json in `kommiunicate` module but it is not accessible in kommunicate module instead it is present in kommunicate ui module. So, we have to go with this approach.

As this function is overloaded we have to are setting the default value to be true so not changing the signature of method. but if the want to mark it as false the they can use new overloaded method.